src/components/PupilViews/PupilReview/PupilReview.view.tsx

```diff
@@ -187,29 +191,32 @@ export const PupilViewsReview = (props: PupilViewsReviewProps) => {
                     $gap={"space-between-s"}
                     $flexDirection={["column", "row"]}
                   >
-                    <OakSecondaryButton
-                      type="button"
-                      role="button"
-                      aria-label="Printable results, opens in a new tab"
-                      title="Printable results (opens in a new tab)"
-                      iconName={"external"}
-                      isTrailingIcon
-                      onClick={handlePrintableResultsClick}
-                      data-testid="printable-results-button"
-                    >
-                      Printable results
-                    </OakSecondaryButton>
+                    <OakBox $display={["none", "flex"]}>
```

Comment:
I don't think you need to wrap this. I think $display is also exposed on OakSecondaryButton ? (might be wrong)

Reply:
i tried that; didnt work